x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -77,118 +89,212 @@ const StyledEuiInMemoryTable = styled(EuiInMemoryTable as any)`
-    eventId={eventId}
-    fieldName={fieldName}
-    fieldType={fieldType}
-    value={value}
-    linkValue={linkValue}
-  />
-);
+  isDraggingDisabled,
+  isRoleSeparator,
+}: SummaryRow['description']) =>
+  values.map((value) => (
+    <FormattedFieldValue
+      key={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}-key`}
+      contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+      eventId={eventId}
+      fieldName={fieldName}
+      fieldType={fieldType}
+      value={value}
+      linkValue={linkValue}
+      isDraggingDisabled={isDraggingDisabled}
+      isRoleSeparator={isRoleSeparator}
+    />
+  ));
 
 const getSummary = ({
   data,
   browserFields,
-  timelineId,
+  timelineId: contextId,
   eventId,
+  isDisplayingThreatSummary,
+  isDisplayingThreatDetails,
 }: {
   data: TimelineEventsDetailsItem[];
-  browserFields: BrowserFields;
+  browserFields?: BrowserFields;
   timelineId: string;
   eventId: string;
+  isDisplayingThreatSummary: boolean;
+  isDisplayingThreatDetails: boolean;
 }) => {
-  return data != null
-    ? fields.reduce<Summary>((acc, item) => {
-        const field = data.find((d) => d.field === item.id);
-        if (!field) {
-          return acc;
-        }
-        const linkValueField =
-          item.linkField != null && data.find((d) => d.field === item.linkField);
-        const linkValue = getOr(null, 'originalValue.0', linkValueField);
-        const value = getOr(null, 'originalValue.0', field);
-        const category = field.category;
-        const fieldType = get(`${category}.fields.${field.field}.type`, browserFields) as string;
-        const description = {
-          contextId: timelineId,
-          eventId,
-          fieldName: item.id,
-          value,
-          fieldType: item.fieldType ?? fieldType,
-          linkValue: linkValue ?? undefined,
-        };
-
-        if (item.id === 'signal.threshold_result.terms') {
-          try {
-            const terms = getOr(null, 'originalValue', field);
-            const parsedValue = terms.map((term: string) => JSON.parse(term));
-            const thresholdTerms = (parsedValue ?? []).map(
-              (entry: { field: string; value: string }) => {
-                return {
-                  title: `${entry.field} [threshold]`,
-                  description: {
-                    ...description,
-                    value: entry.value,
-                  },
-                };
-              }
-            );
-            return [...acc, ...thresholdTerms];
-          } catch (err) {
-            return acc;
+  if (!data) return [];
+  if (isDisplayingThreatSummary || isDisplayingThreatDetails) {
+    return data.reduce<Summary>((acc, { category, field, originalValue }) => {
+      if (isDisplayingThreatDetails && field === 'threat.indicator' && originalValue) {
```

Comment:
We should try to get rid of these magic strings.

Reply:
replaced with INDICATOR_DESTINATION_PATH